game_tic_tac_toe.py

```diff
@@ -159,11 +184,130 @@ def player_won(self, check_disable):
                         {0, 3, 6}, {1, 4, 7}, {2, 5, 8},
                         {2, 4, 6}, {0, 4, 8}]
         for win_set in win_combinations:
-            if win_set.issubset(self.check_disable):
-                for elem in win_set:
-                    self.game_btns[elem]['bg'] = '#CCFFFF'
-                return True
-
+            if win_set.issubset(check_disable):
+                if game_state == 'playing':
+                    for elem in win_set:
+                        self.game_btns[elem]['bg'] = '#CCFFFF'
+                    return True
+                elif game_state == 'trying':
+                    return True
         return False
+    
+    def computer_1turn(self):
+        # first turn is always X
+        # chooses move:
+        corners_center= [0,2,6,8,4]
+        move = random.choice(corners_center)
+        # marks the move, disables the button:
+        self.game_btns[move].config(state="disabled", text='X')
+        # counts a turn: 
+        self.count_turns += 1
+        # adds move to set of moves
+        self.disable_numbers_x.add(move)
+        check_disable = self.disable_numbers_x
+        # defines current and next player
+        current_player = self.player_x
+        next_player = self.player_o
+        self.after_move(check_disable, current_player, next_player)
+
+    def computer_turn(self, comp_symbol):
+        '''This function handles the computer turn '''
+        # chooses move:
+        move = self.deciding_computer_move(comp_symbol)
+        
+        # counts computer turn: 
+        self.count_turns += 1      
+        if comp_symbol == "x":
+            # marks the move, disables the button:
+            self.game_btns[move].config(state="disabled", text="X")
+            # adds move to set of moves
+            self.disable_numbers_x.add(move)
+            check_disable = self.disable_numbers_x
+            # defines current and next player
+            current_player = self.player_x
+            next_player = self.player_o
+        else: 
+            self.game_btns[move].config(state="disabled", text="O")
+            # adds move to set of moves
+            self.disable_numbers_o.add(move)
+            check_disable = self.disable_numbers_o
+            # defines current and next player
+            current_player = self.player_o
+            next_player = self.player_x
```

Comment:
This is one of the places where using arrays for the players and the disabled numbers could safe you about half the lines of code.

Code blocks which are almost the same are always tricky when you edit them later as you might either forget to update one of the blocks or you copy the statement from one block to the other and forget to adapt the variables.